pkgs/by-name/fi/fichub-cli/package.nix

```
@@ -0,0 +1,36 @@
+{ lib, python3Packages, fetchPypi }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "fichub-cli";
+  version = "0.9.4";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-j/+VgiAgJuJJZlBiY/C//cyYELa/+z6pGkqL+Gi8pLE=";
+  };
+
+  nativeBuildInputs = with python3Packages; [
+    setuptools
+  ];
+
+  propagatedBuildInputs = with python3Packages; [
+    platformdirs
+    beautifulsoup4
+    click
+    click-plugins
+    colorama
+    loguru
+    requests
+    tqdm
+    typer
+  ];
+
+  meta = with lib; {
+    description = "A CLI for the fichub.net API";
+    mainProgram = "fichub_cli";
+    homepage = "https://github.com/FicHub/fichub-cli";
+    license = licenses.gpl3;
```

```suggestion
    license = licenses.asl20;
```

https://github.com/FicHub/fichub-cli/blob/v0.9.4/LICENSE.txt